content/docs/start/experiments.md

```diff
@@ -138,6 +111,34 @@ Experiment results have been applied to your workspace.
 
 <details>
 
+### ℹ️ More information about (Hyper)parameters
```

Comment:
This section is in #3163

Reply:
#3163 is merged. Should `master` be merged into this branch so this change goes away, and resolve the comment?